src/components/CompiledOutput.js

```diff
@@ -127,12 +128,38 @@ export function CompiledOutput({
     }
   }
 
+  const sourceCode = compiled?.code ?? "";
   return (
     <Fragment>
       <Grid.Row>
         <Grid.Column width={16}>
           <Menu attached="top" tabular inverted>
             <Menu.Item>input.json</Menu.Item>
+            <Menu.Menu position="left">
+              <Menu.Item>
+                {timeTravel !== null ? (
+                  <Dropdown text="Time Travel">
+                    <Dropdown.Menu>
+                      <Dropdown.Item
+                        text="Source Output"
+                        key={`${sourceCode}`}
+                        onClick={() => setTimeTravelCode(sourceCode)}
+                      />
+
+                      {timeTravel.map(timetravel => (
+                        <Dropdown.Item
+                          text={`${timetravel.currentNode}`}
+                          key={`${timetravel.code}`}
```

Comment:
Why is the code being used as the key?

Reply:
Was to prevent the child components throwing each should have an unique key, however, looking at it now without the `key` doesn't give that error .. will remove!